admin-manual/installation-setup/upgrading/upgrading.rst

```diff
@@ -273,53 +212,71 @@ Upgrade from Archivematica |previous_version| for CentOS/Red Hat
 
-       sudo yum install -y archivematica-devtools
+To complete these instructions you need to know the paths of your transfer
+backlog and AIP storage locations. Respectively, these usually are:
 
-   .. note::
+* :file:`/var/archivematica/sharedDirectory/www/AIPsStore/transferBacklog`
+* :file:`/var/archivematica/sharedDirectory/www/AIPsStore`
 
-      Archivematica devtools is a set of utilities that was built by developers
-      while working on Archivematica. Devtools includes helper scripts that
-      make it easier to perform certain maintenance tasks. One of those tools
-      is used to rebuild the Transfer index in Elasticsearch, which is used by
-      the different backlog tools such as the new Appraisal Tab. Currently this
-      must be installed using git. These instructions will be updated when a
-      packaged version is available.  See the `devtools repo`_ for more details.
+You can confirm the paths of your installation in the Location tab of the
+Storage Service.
 
-2. Find the path to the Transfer Backlog location. The default path is
-   '/var/archivematica/sharedDirectory/www/AIPsStore/transferBacklog'. You can
-   confirm the path for your installation by searching for it in the Storage Service:
+This process has two steps:
 
-   * Log into the Storage Service and click on the Locations tab.
-   * Type 'backlog' in the searchbox.
-   * Copy the value in the column labelled 'path' (there should be only one row)
+1. **Rebuild transfer index:**
 
-3. Using the path you confirmed above, replace the text '/path/to/transfers'
-   with the correct path for your system.
+   Run the following command passing the path of the transfer backlog location
+   you confirmed above:
 
    .. code:: bash
 
-       am rebuild-transfer-backlog /path/to/transfers
+      sudo -u archivematica bash -c " \
+          set -a -e -x
+          source /etc/sysconfig/archivematica-dashboard
+          cd /usr/share/archivematica/dashboard
+          /usr/share/archivematica/virtualenvs/archivematica-dashboard/bin/python manage.py \
+              rebuild_transfer_backlog \
+                  --transfer-backlog-dir=/var/archivematica/sharedDirectory/www/AIPsStore/transferBacklog
+      ";
 
    This may take a while if you have a large backlog. Once it completes, you
    should be able to see your Transfer Backlog in the Appraisal tab and in the
    Backlog tab.
 
+   Depending on your browser settings, you may need to clear your browser cache
+   to make the dashboard pages load properly. For example in Firefox or Chrome
+   yo should be able to clear the cache with ``control-shift-R`` or
+   ``command-shift-F5``.
+
+2. **Rebuild the AIP index:**
+
+   Run the following command passing the path of the AIP storage location you
+   confirmed above:
+
+   .. code:: bash
+
+      sudo -u archivematica bash -c " \
+          set -a -e -x
+          source /etc/sysconfig/archivematica-dashboard
```

Comment:
The`/etc/sysconfig/archivematica-dashboard` config file is only for CentOS. The Ubuntu's file is `/etc/default/archivematica-dashboard`. I think this should be indicated in a note or explicitly for each Distribution.

Reply:
@mamedin can you start a new issue?

Reply:
@sallain done: https://github.com/artefactual/archivematica-docs/issues/139